test/index.js

```diff
@@ -1,34 +1,38 @@
 "use strict"
-var test = require("tap").test
-var requireInject = require("require-inject")
+const tap = require('tap');
+const requireInject = require('require-inject');
 
-test("Windows", function (t) {
-  t.plan(1)
```

Comment:
It looks like the plan was removed -- plans have a lot of utility so I would not remove them without a solid justification.